### DISCUSSION TITLE
How to use multiple engines with on database approach?

### DISCUSSION THREAD

**Discussion by jankoprowski** [sqlalchemy/sqlalchemy#10695] [Usage Questions]
Hello,

  We are straggling issues with pymssql connection to our database. To unlock some functionality we introduced second pyodbc connection to the same database and use them in the part of the project. However changes made via pyodbc are not visible via pymssql and vice versa. Could you please share configuration or method we should use to setup connections to the same database via pymssql and pyodbc in the same project?
  
  Project uses sqlalchmey 1.4

**Comment by CaselIT** [comment]
Hi,

>  However changes made via pyodbc are not visible via pymssql and vice versa. 

I don't follow what you mean by this. After you commit a transaction it's visible by any following transaction, regardless of what driver you are using (and usually it's visible also to existing transaction when using read committed isolation level)

If you are asking "how can I see uncommitted changes from one transaction in another transaction" then you would have to look into read uncommitted isolation level, but you are probably better off avoiding the need to use two different transactions in such cases by avoiding the use of two drivers
